web/satellite/src/components/accessGrants/CreateAccessModal.vue

```diff
@@ -793,7 +794,10 @@ export default class CreateAccessModal extends Vue {
         this.access = accessEvent.data.value;
         await this.$notify.success('Access Grant was generated successfully');
 
+<<<<<<< HEAD
```

Comment:
this will definitely not work, please test Pr after merge confict

Reply:
removed